src/Routing/Adapter/Laravel.php

```diff
@@ -5,8 +5,8 @@
 use Illuminate\Http\Request;
 use Illuminate\Routing\Route;
 use Illuminate\Routing\Router;
-use Illuminate\Routing\RouteCollection;
 use Dingo\Api\Contract\Routing\Adapter;
+use Illuminate\Routing\RouteCollection;
```

Comment:
> Is this change necessary? I think we can revert this.

Same length, it would then order the same length by alphanumeric order, so `D` before `I`.